docs/src/requirements.txt

```diff
@@ -0,0 +1,3 @@
+mkdocs
+mkdocs-material
+mkdocs-redirects
```

Comment:
Missing some requirements here. For example the social plugin.

Reply:
It's not about workflow but about change in `docs/mkdocs.yml`. The social plugin was not used before.